test/integration/serverless/run.sh

```diff
@@ -27,144 +43,141 @@ fi
 
 # Move into the root directory, so this script can be called from any directory
 SERVERLESS_INTEGRATION_TESTS_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
-cd "$SERVERLESS_INTEGRATION_TESTS_DIR/../../.."
 
+cd "$SERVERLESS_INTEGRATION_TESTS_DIR/../../.."
 LAMBDA_EXTENSION_REPOSITORY_PATH="../datadog-lambda-extension"
-
-if [ "$BUILD_EXTENSION" == "true" ]; then
-    echo "Building extension that will be deployed with our test functions"
+if [ "$BUILD_EXTENSION" != "false" ]; then
+    echo "Building extension"
 
     # This version number is arbitrary and won't be used by AWS
     PLACEHOLDER_EXTENSION_VERSION=123
 
     ARCHITECTURE=amd64 VERSION=$PLACEHOLDER_EXTENSION_VERSION $LAMBDA_EXTENSION_REPOSITORY_PATH/scripts/build_binary_and_layer_dockerized.sh
 else
-    echo "Not building extension, ensure it has already been built or re-run with 'BUILD_EXTENSION=true'"
+    echo "Skipping extension build, reusing previously built extension"
 fi
 
-cd "./test/integration/serverless"
-
-# build and zip recorder extension
-echo
-echo "Building recorder extension"
-cd recorder-extension
-GOOS=linux GOARCH=amd64 go build -o extensions/recorder-extension main.go
-zip -rq ext.zip extensions/* -x ".*" -x "__MACOSX" -x "extensions/.*"
-cd ..
+cd $SERVERLESS_INTEGRATION_TESTS_DIR
 
-# build Go Lambda functions
-echo
-echo "Building Go Lambda functions"
-go_test_dirs=("with-ddlambda" "without-ddlambda" "log-with-ddlambda" "log-without-ddlambda" "timeout" "trace")
-cd src
-for go_dir in "${go_test_dirs[@]}"; do
-    env GOOS=linux go build -ldflags="-s -w" -o bin/"$go_dir" go-tests/"$go_dir"/main.go
-done
+./build_recorder.sh
+./build_go_functions.sh
+./build_java_functions.sh
+./build_csharp_functions.sh
```

Comment:
We can now easily comment out these lines if we don't want to rebuild certain functions over and over when testing locally.